doc/api/child_process.md

```diff
@@ -299,6 +299,9 @@ output on this fd is expected to be line delimited JSON objects.
 *Note: Unlike the fork(2) POSIX system call, `child_process.fork()` does
 not clone the current process.*
 
+*Note*: Support for non-Array values of the `.stdio` option was added in
```

Comment:
> Will the FIXME (EDIT: <-- I meant `REPLACEME`) get processed?

Yup, the releaser will be warned by the build script (which does just a plain “grep”) and replace it.

> Our 'ADDED' metadata is not granular enough.

Working on that! ;)